.github/workflows/accuracy_checker.yml

```diff
@@ -1,5 +1,6 @@
 name: github-actions
 on: pull_request
+permissions: read-all  # Required by https://securityscorecards.dev/viewer/?uri=github.com%2Fopenvinotoolkit%2Fopen_model_zoo
```

Comment:
Maybe it doesn't make sense to reference our internal boards in the public repo? just add the permissions line without comments? This is how we do it for other repos.

Reply:
It's public. You can verify that by opening the link using your mobile internet.